external/buildscripts/SDKDownloader.pm

```diff
@@ -176,17 +176,20 @@ sub UpdateSDKRepo
 
 	my $parent_rev = "parent";
 
-	# check if we're in a mercurial repo
-	if (system("hg parent --template \"{node}\"") == 0)
-	{
-		$parent_rev = `hg parent --template "{node}"`;
-	}
-	# check if we're in a git repo
-	elsif (system("git rev-parse HEAD") == 0)
```

Comment:
I've removed this git section completely, as we have no SDKs that use the file (Tizen and STV are the only ones). The git command here will return the last revision of the Mono repo, which is not at all what we want. @UnityAlex can you confirm that this should be fine for STV as well?